tests/music_tests.ts

```diff
@@ -2,27 +2,34 @@
  * VexFlow - Music API Tests
  * Copyright Mohit Muthanna 2010 <mohit@muthanna.com>
  */
+import { KeyManager } from '../src/keymanager';
+import { Music } from '../src/music';
 
-VF.Test.Music = (function () {
-  var Music = {
+declare var QUnit: any;
+declare function ok(...args: any): any;
+declare function test(...args: any): any;
+declare function expect(...args: any): any;
+declare function equal(...args: any): any;
+
+export var MusicTests = (function () {
+  return {
     Start: function () {
       QUnit.module('Music');
-      test('Valid Notes', Music.validNotes);
-      test('Valid Keys', Music.validKeys);
-      test('Note Values', Music.noteValue);
-      test('Interval Values', Music.intervalValue);
-      test('Relative Notes', Music.relativeNotes);
-      test('Relative Note Names', Music.relativeNoteNames);
-      test('Canonical Notes', Music.canonicalNotes);
-      test('Canonical Intervals', Music.canonicalNotes);
-      test('Scale Tones', Music.scaleTones);
-      test('Scale Intervals', Music.scaleIntervals);
+      test('Valid Keys', this.validKeys);
+      test('Note Values', this.noteValue);
+      test('Interval Values', this.intervalValue);
+      test('Relative Notes', this.relativeNotes);
+      test('Relative Note Names', this.relativeNoteNames);
+      test('Canonical Notes', this.canonicalNotes);
+      test('Canonical Intervals', this.canonicalNotes);
+      test('Scale Tones', this.scaleTones);
+      test('Scale Intervals', this.scaleIntervals);
     },
 
     validNotes: function () {
       expect(10);
 
-      var music = new VF.Music();
+      var music = new Music();
```

Comment:
While we're at it I'd change the `var` to `let` or `const`.

Reply:
Done